File: FurnitureStore/UI/furniturestore_mobile/lib/models/payment_order/payment_order.dart
import 'package:json_annotation/json_annotation.dart';

part 'payment_order.g.dart';

@JsonSerializable()
class PaymentOrderModel {
  String? cardNumber;
  String? month;
  String? year;
  String? cvc;
  int? totalPrice;
  PaymentOrderModel(
      this.cardNumber, this.month, this.year, this.cvc, this.totalPrice);

  /// A necessary factory constructor for creating a new User instance
  /// from a map. Pass the map to the generated `_$UserFromJson()` constructor.
  /// The constructor is named after the source class, in this case, User.
  factory PaymentOrderModel.fromJson(Map<String, dynamic> json) =>
      _$PaymentOrderModelFromJson(json);

  // /// `toJson` is the convention for a class to declare support for serialization
  // /// to JSON. The implementation simply calls the private, generated
  // /// helper method `_$UserToJson`.
  Map<String, dynamic> toJson() => _$PaymentOrderModelToJson(this);
}

File: FurnitureStore/FurnitureStore.Models/PaymentOrder/PaymentOrder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FurnitureStore.Models.PaymentOrder
{
   
    public class PaymentOrder
    {
        public string CardNumber { get; set; }
        public string Month { get; set; }
        public string Year { get; set; }
        public string Cvc { get; set; }
        public string CardHolderName { get; set; }
        public int TotalPrice { get; set; }
    }
}
